### DISCUSSION TITLE
Can't install Whisper, stuck after downloading torch-2.0.0-cp38-cp38-manylinux1_x86_64.whl (619.9 MB)

### DISCUSSION THREAD

**Discussion by khalidalmannai** [openai/whisper#1116] [Q&A]
Hi,
It tried to install Whisper using 
pip install -U openai-whisper
and 
pip install git+https://github.com/openai/whisper.git 

**But both of them failed after downloading the "torch" as shown below**
$ pip3 install -U openai-whisper
Collecting openai-whisper
  Using cached openai-whisper-20230314.tar.gz (792 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Collecting numba
  Using cached numba-0.56.4-cp38-cp38-manylinux2014_x86_64.manylinux_2_17_x86_64.whl (3.5 MB)
Collecting tiktoken==0.3.1
  Using cached tiktoken-0.3.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.7 MB)
Collecting ffmpeg-python==0.2.0
  Using cached ffmpeg_python-0.2.0-py3-none-any.whl (25 kB)
Collecting tqdm
  Using cached tqdm-4.65.0-py3-none-any.whl (77 kB)
Collecting triton==2.0.0
  Using cached triton-2.0.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (63.2 MB)
Collecting torch
  Downloading torch-2.0.0-cp38-cp38-manylinux1_x86_64.whl (619.9 MB)
     |████████████████████████████████| 619.9 MB 4.3 MB/s eta 0:00:01Killed

**My system:**
Ubuntu 20.04.6 LTS (GNU/Linux 5.4.0-144-generic x86_64)
Python 3.8.10
pip 20.0.2 from /usr/lib/python3/dist-packages/pip (python 3.8)

Best regards,

**Comment by khalidalmannai** [comment]
I've managed to bypass that problem by using this option --no-cache-dir
pip install -U openai-whisper --no-cache-dir

**Reply by EricTetz** [reply]
Thanks. I was having the same issue trying to install another module ([demucs](https://github.com/facebookresearch/demucs)) and this solved it.
